src/utils/migrate_rolling_stock.py

```diff
@@ -0,0 +1,80 @@
+import json
+import sys
+
+
+COPIED_KEYS_V1_TO_V2 = (
+    "id",
+    "length",
+    "max_speed",
+    "startup_time",
+    "startup_acceleration",
+    "comfort_acceleration",
+    "gamma",
+    "gamma_type",
+    "inertia_coefficient",
+    "features",
+)
+
+
+def migrate_v1_to_v2(old):
```

Comment:
We could add a version field to `RJSRollingStock` and use it for validation.
We have a similar field for `RJSInfra` but we're not using it yet.